static/sass/style.css

```diff
@@ -4000,25 +4000,157 @@ span.highlighted {
 .hidden {
```

Comment:
Just an overall concern of modifying the CSS without updating the relevant SCSS file (styles.scss). This is something we should avoid!

Reply:
ahh sorry, im still not familiar with this SASS stuff 😬

Reply:
Reason: the CSS files here are the result of compiling the SCSS files. we would either need to abandon the SCSS pipeline completely or the next time SCSS is compiled by someone it will blow away these changes.

Reply:
I think ive done this. Reverted all CSS changes in  [097fafc](https://github.com/python/pythondotorg/pull/2593/commits/097fafc5081a76b2216f3910e36eb3c1c4f2b587) then ran the sass compile after i migrated my changes over in [1ab3053](https://github.com/python/pythondotorg/pull/2593/commits/1ab3053e364fab77e2334bd0d94c9a57691fd12b)

Reply:
<img width="351" alt="image" src="https://github.com/user-attachments/assets/e0daa665-4ead-46ee-bcc1-ad8a3000bf20">